adguard-dns.io/eula.html.md

```diff
@@ -171,4 +171,4 @@ Email: [support@adguard.com](mailto:support@adguard.com)
 
 Website: [adguard-dns.io](https://adguard-dns.io)
 
-© 2024 ADGUARD SOFTWARE LIMITED. All Rights Reserved. The Services and any accompanying documentation are copyrighted and protected by copyright laws and international copyright treaties, as well as other intellectual property laws and treaties.
```

Comment:
Please remove the year from these documents, it makes little sense to keep it here and update all the time